src/current/cockroachcloud/plan-your-cluster.md

```diff
@@ -8,163 +8,99 @@ cloud: true
+Storage for a CockroachDB {{ site.data.products.standard }} cluster is allocated on demand and is not covered in this section. Refer to [Storage](#storage-pricing)
 
-## Quick start: Provisioning and adjusting capacity
-<a id="provisioned-capacity"></a>
+### Quickstart: Provision and adjust capacity
 
-Use this section as the easiest way to arrive at the right capacity for your CockroachDB {{ site.data.products.standard }} deployment. Because each workload is different, it is difficult to estimate the required capacity up front. So we recommend starting with a rough estimate and then using the easy scaling capability of Standard to adjust capacity based on workload performance as it evolves.
+To achieve a rough initial estimate of compute capacity, consider:
 
-### Step 1: Estimating initial cluster compute capacity
+- **Expected workload maximum**: the maximum expected long-term compute consumption for the workload, measured in vCPUs.
+  - If your workload is already using CockroachDB {{ site.data.products.core }} and is performing well, we recommend that you start with the sum of the compute capacity of its nodes across all regions.
+  - If you are moving the workload from another database and is performing well, we recommend that you consider its historical compute requirements to arrive at an initial compute capacity.
+  - [Talk to an expert](https://www.cockroachlabs.com/contact/). Engineers at Cockroach Labs will work with you to understand the resource demands of your workload and help you estimate and validate its required capacity.
 
-Follow these steps to achieve a rough initial estimate of compute capacity provisioning for CockroachDB {{ site.data.products.standard }}:
+- **Additional capacity buffer**: To accommodate unanticipated load and mitigate the risk of application degradation if the cluster's capacity is exceeded, we recommend that you plan for additional capacity beyond your initial estimate. A 40% buffer is a good starting point, but more may be needed if the workload is unpredictable or highly sensitive to query latency. Depending on the nature of the workload, such as when demand is seasonal, you can plan to scale the cluster's capacity temporarily to address the workload's requirements.
+  - **Exceeding capacity limit**: CockroachDB {{ site.data.products.standard }} enforces the cluster's provisioned capacity, limiting application throughput if the workload exceeds the cluster's provisioned capacity. This can lead to increased query latency and degraded application performance.
+  - **Notifications to prevent overrun**: To help mitigate against the risk of exceeding a cluster's capacity, email and CockroachDB {{ site.data.products.cloud }} Console notifications will occur when a cluster workload exceeds 90% and 90% of its provisioned capacity.
```

Comment:
```suggestion
  - **Notifications to prevent overrun**: To help mitigate against the risk of exceeding a cluster's capacity, email and CockroachDB {{ site.data.products.cloud }} Console notifications will occur when a cluster workload exceeds 70% and 90% of its provisioned capacity.
```